index.js

```diff
@@ -67,6 +101,13 @@
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 6", "color: red")
 
+// function printNameAndPhones(users) {
+//   for (let i = 0; i < users.length; i++) {
```

Comment:
While a `for` loop is fine for iterating through an array, consider using a `forEach` to practice callbacks!
One possible solution for this would be:
```
users.forEach(function(user){
console.log(user.name)
console.log(user.phones.cell)
console.log(user.phones.office)
})
```

Remember that you can access keys in JavaScript objects via the dot notation, whether they're nested or not!